receiver/oracledbreceiver/metadata.yaml

```diff
@@ -109,6 +109,51 @@ metrics:
       value_type: int
       input_type: string
     unit: "{reads}"
+  oracledb.physical_reads_direct:
```

Comment:
Technically the format and names of these metrics should match [semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/naming/), but we're in a tough spot given existing metrics don't match. (We either match this component's existing metrics, or end up matching semantic conventions and make this component's metrics inconsistent.)

One example that I'm referring to, the semantic convention is to have read and write metrics under the same metric name, with a "direction" attribute that is used to show if the data point is for reads or writes.

I think for now it's okay to add these metrics the way this PR proposes, but we'll likely need a breaking change in the future to fix all of them together.

Reply:
Definitely something to do at another time for all the metrics in one fell swoop (and as a breaking change), if that's what we want.

One advantage of this deviation from the semantic conventions here though is that the metric names match the names from the Oracle documentation. This may help users enable/disable the metrics they care about since they also are OracleDB users if they use this receiver.